### PR TITLE
docs(examples): Using twind v1 + fix syntax highlighting in language

### PR DESCRIPTION
**Pull request by lino-levan**
Went ahead and fixed the syntax highlighting in `setting-the-language` as a drive-by cleanup.

### CODE REVIEW

**Comment by lino-levan** [comment]
ref: https://github.com/denoland/fresh/issues/1220

**Comment by deer** [comment]
> Went ahead and fixed the syntax highlighting in `setting-the-language` as a drive-by cleanup.

thanks

File: docs/examples/using-twind-v1.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+export default {
+  ...defineConfig({
+    presets: [presetTailwind(), presetAutoprefix()],
+  }),
+  selfURL: import.meta.url,
+};
+```
+
+to see what other presets exist, you can go to the
```
Comment (lucacasonato) [inline docs/examples/using-twind-v1.md]:
```suggestion
To see what other presets exist, you can go to the
```

File: docs/examples/using-twind-v1.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import manifest from "./fresh.gen.ts";
+
+import twindPlugin from "$fresh/plugins/twind.ts";
+import twindConfig from "./twind.config.ts";
+
+await start(manifest, { plugins: [twindPlugin(twindConfig)] });
+```
+
+This is great if you want to use twind v0 but if you wanted to use v1, you're
+out of luck. First, you should change the import path to the new twind plugin.
```
Comment (lucacasonato) [inline docs/examples/using-twind-v1.md]:
```suggestion
The template generates a Twind v0 project by default. If you want to use Twind v1 you can follow this guide. First of all, change the import path to use the `twindv1` plugin:
```

**Comment by lino-levan** [comment]
@marvinhagemeister Please take a look!

**Review by marvinhagemeister — APPROVED** [review]
This is great! Thank you so much for taking the time to write this down 🎉

**Comment by J053Fabi0** [comment]
The example code gives an error:

```
error: TS2345 [ERROR]: Argument of type '{ selfURL: string; darkMode?: DarkModeConfig; darkColor?: DarkColor<BaseTheme & TailwindTheme & { screens: Record<string, MaybeArray<ScreenValue>>; colors: Record<...>; }> | undefined; ... 7 more ...; finalize: Finalize<...>[]; }' is not assignable to parameter of type 'Options'.
  Types of property 'darkColor' are incompatible.
    Type 'DarkColor<BaseTheme & TailwindTheme & { screens: Record<string, MaybeArray<ScreenValue>>; colors: Record<string, MaybeColorValue>; }> | undefined' is not assignable to type 'DarkColor<BaseTheme> | undefined'.
      Type 'DarkColor<BaseTheme & TailwindTheme & { screens: Record<string, MaybeArray<ScreenValue>>; colors: Record<string, MaybeColorValue>; }>' is not assignable to type 'DarkColor<BaseTheme>'.
        Types of parameters 'context' and 'context' are incompatible.
          Type 'Context<BaseTheme>' is not assignable to type 'Context<BaseTheme & TailwindTheme & { screens: Record<string, MaybeArray<ScreenValue>>; colors: Record<string, MaybeColorValue>; }>'.
            The types returned by 'theme()' are incompatible between these types.
              Type 'BaseTheme' is not assignable to type 'BaseTheme & TailwindTheme & { screens: Record<string, MaybeArray<ScreenValue>>; colors: Record<string, MaybeColorValue>; }'.
await start(manifest, { plugins: [twindPlugin(twindConfig)] });
```

I just created a project running `deno run -A -r https://fresh.deno.dev` and copy-paste the contents exactly as they are in the example for `main.js` and `twind.config.ts`.

**Pull request merged**